lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.8.0-SNAPSHOT",
+  "version": "2.9.0-SNAPSHOT",
```

Comment:
Should this be `2.8.1-SNAPSHOT`?

Reply:
addressed in commit https://github.com/zowe/vscode-extension-for-zowe/pull/2267/commits/4a68c1a4b4fcbe9a304ad8014a08d21163f4a2fe